MapleServer2/Commands/Game/InfoCommands.cs

```diff
@@ -29,13 +29,22 @@ public override void Execute(GameCommandTrigger trigger)
             {
                 foreach (string alias in command.Aliases)
                 {
-                    stringBuilder.Append(CommandHelpers.Color(CommandHelpers.Bold(alias), Color.DarkOrange));
+                    stringBuilder.Append(CommandHelpers.Color(CommandHelpers.Bold(alias), Color.DarkOrange) + " ");
                 }
 
                 stringBuilder.Append($"{CommandHelpers.Color(command.Description, Color.Wheat)}\n");
+                //if (command != commandList.Last())
+                //{
+                //    stringBuilder.Append('\n');
+                //}
```

Comment:
Can this be removed if its just commented out?

Reply:
sorry ... I didn't see it 🙈